src/mach/symbols.rs

```diff
@@ -405,18 +405,11 @@ impl<'a> Symbols<'a> {
 
 impl<'a> Debug for Symbols<'a> {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-        writeln!(fmt, "Data: {} start: {:#?}, nsyms: {} strtab: {:#x}", self.data.len(), self.start, self.nsyms, self.strtab)?;
-        writeln!(fmt, "Symbols: {{")?;
-        for (i, res) in self.iter().enumerate() {
-            match res {
-                Ok((name, nlist)) => {
-                    writeln!(fmt, "{: >10x} {} sect: {:#x} type: {:#02x} desc: {:#03x}", nlist.n_value, name, nlist.n_sect, nlist.n_type, nlist.n_desc)?;
-                },
-                Err(error) => {
-                    writeln!(fmt, "  Bad symbol, index: {}, sym: {:?}", i, error)?;
-                }
-            }
-        }
-        writeln!(fmt, "}}")
```

Comment:
Should we keep this? And how?

Reply:
The elf symtab [prints them as a vec](https://github.com/m4b/goblin/blob/4c24bc5a615960b99a6eb80b36c34497f99ece96/src/elf/sym.rs#L493). We use this as a lazy way of implementing the `rdr` example, and it's handy for it to print out the symbols too, so I think we should keep it.

Reply:
Fixed.

Reply:
Looking again, we don't actually use this for the `rdr` example (`MachO` creates a vec for its symbols instead), but I think this is still fine.